### DISCUSSION TITLE
buddhism/2024/projects/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#135] [General]
# buddhism/2024/projects/

https://shanenull.com/buddhism/2024/projects/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2024/heart_sutra_memorization/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/command_line_tools/

**Comment by shane0** [comment]
# same click app

- wip screencast coming

**Reply by shane0** [reply]
Add these ten with a node decision tree
https://youtu.be/MFkfl-1NcqM
